### ISSUE TITLE
ensure that user testing is incorporated into week 10 & 12 in all locations

### ISSUE THREAD

**Issue by jsms90** [foundersandcoders/master-reference#577]
### London
FAC10 had user testing baked into their week 10 with @harrydry @nogainbar & @bradreeder. Some did this with alumni upstairs, some went outside.
Then @njsfield held a user testing session on the last day of week 12.

[FAC10 week 10 SGC](https://github.com/foundersandcoders/london-curriculum/blob/master/stop-go-continue/fac-10/cohort-week-10.md#continue)
> #### Continue
> + User testing
> + Outside user testing was very helpful

### Nazareth
As far as I know, FACN1 didn't have user testing in week 10. Apparently they code reviewed each other's apps, but didn't do actual interviews.
I did a user testing session with them on the last day of week 12.

[FACN1 week 10 SGC](https://github.com/foundersandcoders/nazareth-curriculum/blob/master/stop-go-continue/fac-n1/week-11.md#time-to-improve-the-app)
> #### Go
> + More time for discussion and critique of prototypes
>    + Include some sort of user testing or something in class

[FACN1 week 12 SGC](https://github.com/foundersandcoders/nazareth-curriculum/blob/master/stop-go-continue/fac-n1/week-13.md#time-to-improve-the-app)
> #### Go
> + Time to improve the app
>    + It feels frustrating to do user testing and find out all this stuff that you could improve on and then do nothing with it. Doesn't that kind of lose the point of user testing a bit?
>    + If there's user testing at the end of week 10 that we iterate upon over week 11-12, then maybe it's not necessary after all?
>
> #### Continue
> + Outside user testing
>    + More time on user testing!

Gaza may struggle more with the requirement to go outside and talk to people on the street for user testing (?) but they could still do it with each other.

**Comment by eliascodes** [comment]
@jsms90 in Gaza, going outside is fine for the students, but I don't think it would be a fruitful exercise. Much better would be to try to make use of the other people already in the GSG space; get some of the people at the start-ups or on the GSG staff to help user test.

**Comment by mattlub** [comment]
> Then @njsfield held a user testing session on the last day of week 13.

What was this? What was being user-tested or was it instructional?

**Comment by jsms90** [comment]
@eliascodes Fair enough. I don't know what it's like in Gaza. But in London & Nazareth, I would definitely recommend that they don't user test on each other. You don't get nearly as much out of an interview with those who make apps for a living vs "normal" users.

**Comment by jsms90** [comment]
@njsfield could you PR in [your gist](https://gist.github.com/njsfield/cad2904e0fa3a74b32fd5accbca50c2e) from the last day of the build sprint?

It could go directly into weeks 10-12 like:
[coursebook/weeks-10-12/user-testing.md](https://github.com/foundersandcoders/master-reference/tree/master/coursebook/weeks-10-12)

**Comment by zurda** [comment]
Is there an action item for this, or was this already resolved? Happy to help if needed!

**Comment by m4v15** [comment]
I think it's kind of resolved/outdated - @arrested-developer I think did a lot of user testing in London for FAC15, and I assume @charlielafosse has something similar planned. I _think_ there is already enough divergence and this is more of CF led thing that ensuring uniformity across campuses is unnecessary at this stage

**Comment by arrested-developer** [comment]
Now that weeks 10-16

**Comment by arrested-developer** [comment]
oops! clumsy fingers...

Not sure if this is an issue for `master-reference` or `london-programme`

User testing should now be well embedded in the weeks 10-18 timetables in `london-programme` but this kind of thing can always be improved. I do find that a lot of the material in weeks 10+ is not that thoroughly resourced and depends a lot on who is in the room to deliver it. IMO it might be worth thinking about how that part of the course can be made more self-sufficient / less mentor-dependent as design week can be hard to staff with experienced UX people.

**Comment by charlielafosse** [comment]
Agree with what @arrested-developer is saying... we're currently in the midst of Week 10 now so will be back soon with some more feedback and thoughts :seedling:
